app/2.0/start/install-2-0.md

```diff
@@ -4,107 +4,96 @@ title: Install Polymer 2.x
 
 <!-- toc -->
 
-You can install Polymer via the Polymer CLI, or via Bower.
+Polymer is distributed via the [Bower package manager](https://bower.io/). 
 
-### Option 1:  Start a blank application with `polymer init`
+To create an application template and install Polymer automatically, you can [use the Polymer CLI](#use-cli).
 
-1. Install the Polymer CLI
+To start a project from scratch, you can [install Polymer with Bower](#use-bower). 
+
+### Use the Polymer CLI to create an application template and install Polymer {#use-cli}
+
+1. Install Bower.
+
+    ```bash
+    npm install -g bower
+    ```
+
+2. Install the Polymer CLI
 
     ```bash
     npm install -g polymer-cli@next
```

Comment:
I think we won't need the @next by the time we deploy these docs, so we should take a pass and remove them all.